dts/bindings/pinctrl/wch,afio.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The AFIO peripheral is used to configure pin remapping, EXTI sources and,
+  when available, enable the I/O compensation cell.
+
+compatible: "wch,afio"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+  "#address-cells":
+    required: true
+    const: 1
+  "#size-cells":
+    required: true
+    const: 1
+
+child-binding:
+  description: |
+    Each child node defines the configuration for a particular state.
+  child-binding:
+    description: |
+      The grandchild nodes group pins that share the same pin configuration.
+
+    include:
+      - name: pincfg-node.yaml
+        property-allowlist:
+          - bias-high-impedance
+          - bias-pull-up
+          - bias-pull-down
+          - drive-open-drain
+          - drive-push-pull
+
+    properties:
+      slew-rate:
+        type: string
+        default: "max-speed-2mhz"
+        enum:
+          - "max-speed-10mhz"
+          - "max-speed-2mhz"
+          - "max-speed-50mhz"
```

Comment:
s/50mhz/30mhz/

Reply:
What? It should be 50 - not that it's "actually" 50 but it's design is 50.

Reply:
"7.3.1.1 Port Configuration Register Low" of  CH32V003RM v1.7 says that the output modes are 10 MHz, 2 MHz, and 30 MHz.

Reply:
This should be fixed now - thanks!